docs/_sources/usage.rst.txt

```diff
@@ -64,6 +64,42 @@ First, setup Homebrew [#homebrew]_ and afterwards install the RTRlib package:
 
 .. [#homebrew]  Homebrew -- http://brew.sh
 
+
+Gentoo
+------
+
+The kind guys from the FRRouting project maintain a gentoo overlay [#overlay]_ that contains an ebuild for the RTRlib.
+First, setup layman [#layman]_, then install rtrlib with the following commands:
+
+.. code-block:: bash
+
+    # If this doe not work try layman -f
+    layman -a frr-gentoo
+    emerge rtrlib
+
+.. rubric:: Footnotes
+
+.. [#overlay] https://github.com/FRRouting/gentoo-overlay
+.. [#layman] https://wiki.gentoo.org/wiki/Layman
+
+
+Debian
+------
+
+Debian contains RTRlib since Buster [#DebianBuster]_ and can be installed with apt.
+The package is called ``librtr0``, developers will require the ``librtr-dev`` package, too.
+
+.. code-block:: bash
+
+    apt install librtr0 librtr-dev
```

Comment:
Like described for the gentoo part, it could be added how to add the buster repository to your apt sources list

Reply:
That is not comparable. For gentoo it is an additional repository compatible with mainline gentoo. For Debian you would have to include the repository of a completely different Debian version, which you absolutely do not want to do. Unless you want to upgrade.